### ISSUE TITLE
Clean up preferences

### ISSUE THREAD

**Issue by kathyccheng** [bloom-housing/bloom#2334]
Preference ID, Name, Descriptions, Sub Options + Descriptions, Listings it's associated with 

* Remove duplicate preferences 
* Assign missing preferences again 
* See if we need to retroactively assign 

Which preferences to remove
https://docs.google.com/spreadsheets/d/1ZwXVSuiqmN7WiLOMofhGzL6fwCsfDGKz6DCpYFCLfy0/edit#gid=74966850

Correct Preferences per Listing 
https://docs.google.com/spreadsheets/d/1mJzY4vv1V4WkyYjU7R7aWLhHGaWlgDLKj5QM1i27DlI/edit#gid=0

**Comment by kathyccheng** [comment]
@seanmalbert I added a spreadsheet for which preferences to keep / remove / update a name

For preferences that we're removing, can we reassign existing listings to the de-duplicated one we're keeping?

**Comment by seanmalbert** [comment]
@kathyccheng,

I cleaned up the preferences as indicated with https://docs.google.com/spreadsheets/d/1ZwXVSuiqmN7WiLOMofhGzL6fwCsfDGKz6DCpYFCLfy0/edit#gid=74966850. I marked the ones that had listings reassigned in yellow and updated the notes to show which listings are currently assigned. Here they are, ordered by preference title:
![Screen Shot 2022-01-26 at 8 37 43 PM](https://user-images.githubusercontent.com/4401157/151288056-b9156ceb-78b2-4341-b22e-b1164c1497ed.png)

I'm not sure I quite follow the data in https://docs.google.com/spreadsheets/d/1mJzY4vv1V4WkyYjU7R7aWLhHGaWlgDLKj5QM1i27DlI/edit#gid=0, so I haven't done anything with that yet.
